src/App/App.tsx

```diff
@@ -39,6 +40,10 @@ function App() {
 								</Typography>
 							</Container>
 						</Route>
+						<Route path="/searchUsers" exact>
```

Comment:
Can we update the URL route here to use a dash and lower case? I loathe mixed case URLs >.>

```suggestion
						<Route path="/search-users" exact>
```